scheduler-server/src/main/java/gov/nasa/jpl/aerie/scheduler/server/services/TypescriptCodeGenerationService.java

```diff
@@ -86,11 +86,35 @@ private static String generateActivityTemplateConstructors(final Iterable<Activi
     final var result = new ArrayList<String>();
     result.add("const ActivityTemplateConstructors = {");
     for (final var activityTypeCode : activityTypeCodes) {
-      result.add(indent("%s: function %sConstructor(args: {".formatted(activityTypeCode.activityTypeName(), activityTypeCode.activityTypeName())));
-      result.add(indent(indent(generateActivityArgumentTypes(activityTypeCode.parameterTypes()))));
-      result.add(indent("}): %s {".formatted(activityTypeCode.activityTypeName())));
-      result.add(indent(indent("return { activityType: ActivityType.%s, args };".formatted(activityTypeCode.activityTypeName()))));
-      result.add(indent("},"));
+      if(activityTypeCode.parameterTypes().size() == 0) {
```

Comment:
Prefer `.isEmpty()` to `.size() == 0`
```suggestion
      if(activityTypeCode.parameterTypes().isEmpty()) {
```

Reply:
Fixed!